pkg/tnf/handlers/liveness/liveness.gotemplate

```diff
@@ -0,0 +1,3 @@
+{{- range .Containers.test.Liveness -}}
```

Comment:
if platform.preStop will not verify if liveness exist.
please update the template to check for liveness and not preStop